src/clarity-angular/popover/signpost/_signposts.clarity.scss

```diff
@@ -12,20 +12,24 @@
             cursor: pointer;
         }
 
-        .signpost-action {
-            min-width: $clr_baselineRem_1;
-            margin: 0;
-            padding: 0;
+        .signpost-trigger {
             color: $clr-dark-midtone-gray;
-            outline: none;
+            border: none;
 
-            clr-icon {
-                width: $clr_baselineRem_1;
-                height: $clr_baselineRem_1;
+            &:active {
+                border: none;
+            }
+
+            &.active, &:hover {
+                color: $clr-action-blue;
+                background-color: transparent;
             }
 
-            &:hover, &.active {
+            &:focus {
                 color: $clr-action-blue;
+                outline: 0;
```

Comment:
@hippee-lee: Sorry but I just remembered that UX wants our outline styles to look the same across all our components. I think you should confirm with UX on how the outline should look. If I remember correctly, outline styles should match that of regular buttons so you would have to remove `outline: 0` and the border styles below and just let the browser add the outline styles.